source/Octopus.Client/Model/OverwriteMode.cs

```diff
@@ -10,5 +12,16 @@ public enum OverwriteMode
     internal static class OverwriteModeLink
     {
         public static string Link = "overwriteMode";
+
+        public static bool AsLegacyReplaceFlag(this OverwriteMode @this, ILog logger)
```

Comment:
Consider a more explicit name like this? I felt more confident of understanding the intent of the code as I imagined a name like this:
```suggestion
        public static bool ConvertToLegacyReplaceFlag(this OverwriteMode @this, ILog logger)
```